
/* re_common
 * Copyright (C) 2016-2017 The University of Adelaide
 *
 * This file is part of "re_common"
 *
 * "re_common" is free software: you can redistribute it and/or
 * modify it under the terms of the GNU Lesser General Public
 * License as published by the Free Software Foundation; either
 * version 3 of the License, or (at your option) any later version.
 *
 * "re_common" is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 * Lesser General Public License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public
 * License along with this library.  If not, see
 * <http://www.gnu.org/licenses/>.
 */

syntax="proto3";

package EnvironmentControl;

message ShutdownExperimentRequest{
	string experiment_name = 1;
	bool is_regex = 2;
}

message ShutdownExperimentReply{
	repeated string	experiment_names = 1;
}

message ListExperimentsRequest{

}

message ListExperimentsReply{
	repeated string	experiment_names = 1;
}

message GetQpidBrokerEndpointRequest{

}

message GetQpidBrokerEndpointReply{
	string endpoint = 1;
}

message GetTaoCosnamingEndpointRequest{

}

message GetTaoCosnamingEndpointReply{
	string endpoint = 1;
}